controllers/route.go

```diff
@@ -178,6 +180,8 @@ func newTemplateParams(r *http.Request) templateParams {
 // Base handles the default path and template execution
 func (as *AdminServer) Base(w http.ResponseWriter, r *http.Request) {
 	params := newTemplateParams(r)
+
+	fmt.Println(params)
```

Comment:
Looks like we can delete this debug line

Reply:
Ah whoops, good catch. Removed.